COPYING.md

```diff
@@ -0,0 +1,25 @@
+# License
+
+Licensed under either of
+
+ * Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or http://www.apache.org/licenses/LICENSE-2.0)
+ * MIT license ([LICENSE-MIT](LICENSE-MIT) or http://opensource.org/licenses/MIT)
+
+at your option.
+
+# Contribution
+
+Unless you explicitly state otherwise, any contribution intentionally
+submitted for inclusion in the work by you, as defined in the Apache-2.0
+license, shall be dual licensed as above, without any additional terms or
+conditions.
+
+# Note on license change
+
+The licensing of this crate changed from the Bootstrap Open Source License
+to the above "MIT OR Apache-2.0" dual license in release 0.6.1. The license
```

Comment:
There are breaking changes already merged into the repo, so the next crate version will be 0.7.0:
```suggestion
to the above "MIT OR Apache-2.0" dual license in release 0.7.0. The license
```